prototype/gmail_accounts.py

```diff
@@ -0,0 +1,101 @@
+import string
+from httplib2 import Http
+from random import choice
+
+from apiclient.errors import HttpError
+from apiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+PASSWORD_CHARS = string.ascii_letters + string.digits
+
+def generate_password(length=10):
```

Comment:
There's a `get_random_string` function in `django.utils.crypto`, maybe we can reuse that?